pyfar/dsp/filter/band_filter.py

```diff
@@ -461,7 +461,8 @@ def notch(signal, center_frequency, quality, sampling_rate=None):
     (high quality factor). It rejects a narrow frequency band around the
     center frequency with a gain of 0 (:math:`-\\infty` dB) at the center
     frequency and leaves the rest of the spectrum little changed
-    with gains close to 1 (0 dB). Wrapper for :py:func:`scipy.signal.iirnotch`.
+    with gains close to 1 (0 dB) [#]_. Wrapper for
+    :py:func:`scipy.signal.iirnotch`.
```

Comment:
🍺

Reply:
Sphinx8 returns a warning here and Sphinx7 isnt, but ok cheers

Reply:
aha, maybe a small one...